.github/workflows/real-time-ensembler.yaml

```diff
@@ -0,0 +1,109 @@
+name: engines/real-time-ensembler
+
+on:
+  # Automatically run CI on Release and Pre-Release tags and main branch
+  # (only if there are changes to relevant paths)
+  push:
+    tags:
+      - "real-time-ensembler/v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
nit: I understand that you followed the naming convention of `batch-ensembler`, hence picked up the `real-time-ensembler` name for this module. Maybe we should consider changing this to `pyfunc-ensembler-job` and `pyfunc-ensembler-service` for a better representation of what it does?

Reply:
Ok those parts have been refactored; all mentions of `batch-ensembler` or `real-time-ensembler`s in the entire turing repo have been worked around to their new names/references `pyfunc-ensembler-job` and `pyfunc-ensembler-service` respectively.